app/controllers/shopping_list_items_controller.rb

```diff
@@ -27,7 +27,7 @@ def list_item_params
     params.require(:shopping_list_item).permit(
       :description,
       :quantity,
-      :notes
+      :notes,
```

Comment:
I don't like this style but it's more annoying to add an item to the end of a list, forget to add a comma to what was previously the last element, and get an error.